### ISSUE TITLE
xmlns declarations fail if Assembly starts with capital letter

### ISSUE THREAD

**Issue by aguahombre** [AvaloniaUI/Avalonia#2837]
Using the XAMLIL compiler in build 0.8.999-cibuild0003548-beta I get unable to resolve type error if the xmlns declaration has Assembly starting with a capital letter.
This used to work in v0.8.1.

````
<UserControl xmlns="https://github.com/avaloniaui"
             xmlns:x="http://schemas.microsoft.com/winfx/2006/xaml"
             xmlns:d="http://schemas.microsoft.com/expression/blend/2008"
             xmlns:mc="http://schemas.openxmlformats.org/markup-compatibility/2006"
             xmlns:local="clr-namespace:MyProject;Assembly=MyProject"
            x:Class="MyProject.Views.MyView">

  <UserControl.Resources>
    <local:MyConverter x:Key="MyConverter"/>
  </UserControl.Resources>

````

This now gives a compile error:
```
Error	XAMLIL	Unable to resolve type MyConverter from namespace clr-namespace:MyProject;Assembly=MyProject 
```

If I modify the line with lowercase assembly , the error goes away
```
xmlns:local="clr-namespace:MyProject;assembly=MyProject"
```

Is this expected behaviour or a bug?

**Comment by jmacato** [comment]
@aguahombre  i think it's the same behavior as WPF? it's by convention that both `clr-namespace` and `assembly` are all lowercase iirc.

**Comment by aguahombre** [comment]
If lowercase assembly is required, the XAMLIL compiler should output an error for the xmlns source line with a meaningful explanation?

**Comment by jmacato** [comment]
@aguahombre yeah i agree that it should, perhaps @kekekeks could add that error (if its btw) message.

**Comment by aguahombre** [comment]
Just tried Assembly in WPF and the designer underlines the xmlns statement in blue and shows a tooltip message
![image](https://user-images.githubusercontent.com/17591364/62929629-c1841d00-bdb2-11e9-8469-1c69b2d54b9f.png)

**Comment by danwalmsley** [comment]
@aguahombre we currently dont have that luxury, (to show a warning / error in ide)

we only get to find this out at compile time. However the error message could be clearer.
